docs/cli.md

```diff
@@ -218,6 +219,16 @@ If this option is enabled, the following values are used as cache keys and the f
 prettier --write --cache src
 ```
 
+Running Prettier without `--cache` will delete the cache.
+
+Also, since the cache file is stored in `./node_modules/.cache/prettier/.prettier-cache`, so you can use `rm -f . /node_modules/.cache/prettier/.prettier-cache` to remove them manually.
+
+:::caution
```

Comment:
Is this a emoji?

Reply:
Ah sorry docusauls v1 does not support https://docusaurus.io/docs/markdown-features/admonitions

Reply:
[0f57b23](https://github.com/prettier/prettier/pull/12800/commits/0f57b2326750cbacc471efc5f204794f13b8f7f4)